src/main/scala/com/horizon/exchangeapi/OrgsRoutes.scala

```diff
@@ -327,6 +340,69 @@ trait OrgsRoutes extends JacksonSupport with AuthenticationSupport {
       }) // end of complete
     }   // end of exchAuth
   }
+  // ====== GET /orgs/{orgid}/status ================================
+  @GET
+  @Path("orgs/{orgid}/status")
+  @Operation(summary = "Returns an org specific status info", description = "Returns the org with the specified id. Can be run by any user in this org or a hub admin.",
```

Comment:
- The `summary` should be: Returns summary status of the org
- The `description` should be: Returns the totals of key resources in the org. Can be run by any id in this org or a hub admin.